save-common/src/jvmMain/kotlin/org/cqfn/save/entities/Execution.kt

```diff
@@ -14,7 +14,9 @@ import javax.persistence.ManyToOne
  * @property endTime
  * @property status
  * @property testSuiteIds
+ * @property offset
  * @property resourcesRootPath path to test resources, relative to shared volume mount point
+ * @property executionLimit
```

Comment:
Pls write a couple of words here, this name is not obvious

Reply:
Fixed